### PR TITLE
Fix password reset issue

### PR DESCRIPTION
**Pull request by ritmatter**
This should fix the password reset.

I couldn't actually find anything wrong, except that the formatting on the input box was really messed up. I restyled that and caught some other weird things along the way

### CODE REVIEW

File: api/routes/UserRoutes.js [hunk 1]
```diff
@@ -277,12 +277,15 @@ module.exports = function(router, isAuthenticated, acl, censor) {
...
+      // Remove the password reset token
       PasswordResetToken.findOneAndRemove({ 'string': token }, function(err, passwordResetToken) {
         if (err) {
           winston.log('error', 'Error finding password reset token: %s', err);
           return res.status(501).send(err);
+
+        // Check if the password reset token was not found or it is has no user
         } else if (!(passwordResetToken && passwordResetToken.user)) {
-          winston.log('error', 'The user provided (%s) was invalid.', email);
+          winston.log('error', 'Password reset token was not found for token: %s', token);
```
Comment (ritmatter) [inline api/routes/UserRoutes.js]: email is not a variable here

File: js/components/PasswordReset.jsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ var PasswordReset = React.createClass({
     }
 
     return (
-      <div className="col-md-2 col-md-offset-10 password-reset">
+      <div className="password-reset">
```
Comment (ritmatter) [inline js/components/PasswordReset.jsx]: col-md-2 inside another col-md-2 due to .login-form

File: js/components/PasswordResetPage.jsx [hunk 3]
```diff
@@ -35,7 +35,7 @@ var PasswordResetPage = React.createClass({
         this.transitionTo('/home');
       }.bind(this),
       error: function(xhr, status, err) {
-        console.log(xhr);
+        console.log(xhr.responseText);
```
Comment (ritmatter) [inline js/components/PasswordResetPage.jsx]: xhr is a big option, we want the responseText

**Comment by krazemon** [comment]
LGTM

**Pull request merged**